lib/stink_bomb/configuration.rb

```diff
@@ -6,8 +6,31 @@ def initialize(options = {})
       self.message = options.fetch(:message) { default_message }
     end
 
+    def raise=(error_class)
+      case error_class
+      when true
+        add_bomb(RaiseBomb.new)
+      when false
```

Comment:
Yeah, it will add twice. I was wondering if there was a use case for wanting to add two errors, but probably not. The other option would be to move `remove_bomb` to the top and just add bombs here:

```ruby
remove_bomb(RaiseBomb)
if error_class == true
  add_bomb(RaiseBomb.new)
elsif error_class
  add_bomb(RaiseBomb.new(error_class: error_class)
end
```